spec/rails_helper.rb

```diff
@@ -0,0 +1,57 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
```

Comment:
Style/ExpandPathArguments: Use `expand_path('../config/environment', __dir__)` instead of `expand_path('../../config/environment', __FILE__)`.